certified-connectors/dox42/apiDefinition.swagger.json

```diff
@@ -56,6 +56,53 @@
             "x-ms-summary": "Basic accept header, leave as is!"
           }
         ]
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@dox42diegoschleis empty response schemas and default responses with an empty schema are not allowed. Could you please define successful 2xx response and a schema? An example of responses can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
I have now defined a 200 response, our service will not return 4xx or 5xx by default, so defining them is out of the question

Reply:
Could you please change the default response to 200 response and remove an empty schema in this case? If you want to keep the schema, you can define it like string type in your other operations.

Reply:
Changed